### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by lyrmin** [dmustanger/7dtd-ServerTools#343]
Hi. I have some problems. It is very hard to localize manualy files.
Old file: https://prnt.sc/1xrgpe7
New file: https://prnt.sc/1xrgrtg
It is possibe that i'm create file with localization, wich will be used by ServerTools?
Maybe it is already in ServerTools?

**Comment by ObsComp** [comment]
Yes you can set any language you want to use. Replace the servertools responses to whatever you want them to be. That is why it is available in that manner. Once you have done it, copy the file so if you ever need to put the replacement entries back in, you can with ease

**Comment by ObsComp** [comment]
Phrases.xml controls all of the responses. Change them to anything you want. CommandList.xml controls the /commands players must use to trigger tools. You can replace all of these with any language you prefer

**Comment by ObsComp** [comment]
There is no generic localization option unfortunately

**Comment by lyrmin** [comment]
Thank you!
